src/services/ledger/facade.ts

```diff
@@ -64,7 +64,7 @@ export const recordSend = async ({
   return persistAndReturnEntry({ entry, hash: metadata.hash })
 }
 
-export const recordReceive = async ({
+export const recordReceiveContraLnd = async ({
```

Comment:
I did this in payout PR (https://github.com/GaloyMoney/galoy/pull/2684/commits/4917f6e874d9de8dabaff37e813130a9cf8e1c0e) because all offchain went against lnd account, but in this PR there's still some onchain places where we're writing things against the lnd account.

But now I'm thinking, we'll eventually import lnd xpubs to bria as well anyway, so maybe we should change all receives (lnd and bria) to be agains that new `OnChain` accounting path now?

Reply:
Resolved with new handling https://github.com/GaloyMoney/galoy/pull/2662/commits/9a914bc45b9ef0a46d62f75f4636a801a1c8037d